sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/spatial/GeometryDeserializer.java

```diff
@@ -1,18 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.core.util.serializer;
-
-import com.azure.core.models.spatial.CollectionGeometry;
-import com.azure.core.models.spatial.Geometry;
-import com.azure.core.models.spatial.GeometryBoundingBox;
-import com.azure.core.models.spatial.GeometryPosition;
-import com.azure.core.models.spatial.LineGeometry;
-import com.azure.core.models.spatial.MultiLineGeometry;
-import com.azure.core.models.spatial.MultiPointGeometry;
-import com.azure.core.models.spatial.MultiPolygonGeometry;
-import com.azure.core.models.spatial.PointGeometry;
-import com.azure.core.models.spatial.PolygonGeometry;
+package com.azure.core.experimental.spatial;
```

Comment:
There's a `com.azure.core.experimental.serializer` too. Maybe we should put this in `com.azure.core.experimental.spatial.serializer`?

Reply:
Placed the serializers in spatial to denote they are tied to the experimental concept of spatial and not serializers.